apps/codeforafrica/src/lib/index.js

```diff
@@ -1537,6 +1537,42 @@ function getStoryPageStaticProps(params) {
   return { notFound: true };
 }
 
+function getAboutMembersPageStaticProps() {
+  return {
+    props: {
```

Comment:
@kilemensi  all this will be moved to CMS right?

Reply:
Yes @esirK !